Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "elinvar"
+version = "0.0.1"
+authors = [
+    "Graham Cox <graham@grahamcox.co.uk>"
```

Comment:
Absolutely. I didn't know the proper values for everyone who would be involved, so I figured people could do it as they go